src/core/load_balancing/rls/rls.cc

```diff
@@ -690,6 +755,13 @@ class RlsLb : public LoadBalancingPolicy {
   // Updates the picker in the work serializer.
   void UpdatePickerLocked() ABSL_LOCKS_EXCLUDED(&mu_);
 
+  void MaybeExportPickCount(
+      GlobalInstrumentsRegistry::GlobalUInt64CounterHandle handle,
+      absl::string_view target, const PickResult& pick_result);
+
+  const std::string instance_id_;
+  std::unique_ptr<RegisteredMetricCallback> registered_metric_callback_;
```

Comment:
It looks like the `mu_` will be destroyed before the callback. That's going to result in a crash.